### DISCUSSION TITLE
CameraPreview Control for Avalonia

### DISCUSSION THREAD

**Discussion by CastelloBrancoTecnologia** [AvaloniaUI/Avalonia#12745] [Q&A]
I need to take a picture of Android/IOS (like CameraPreview control) and also capture an QRCode.

Apparently Avalonia.Essentials is stagnant and there is no WindowsCommunityToolkit with the CameraPreview control.

Does anyone have a path to show me?

Thanks

**Comment by timunie** [comment]
You can try to use MAUI.Essentials in combination with Avalonia. Should work in general.
